### ISSUE TITLE
Missing Material Icons

### ISSUE THREAD

**Issue by ksankumar** [flutter/flutter#56046]
I noticed that some material icons are missing from Flutter including sports_cricket.

**Comment by Abhishek01039** [comment]
you can use this https://pub.dev/packages/font_awesome_flutter

**Comment by HansMuller** [comment]
CC @willlarche

**Comment by willlarche** [comment]
Waiting on #49737 being opt-out. Currently an issue with web blocking us right now.

**Comment by TahaTesser** [comment]
duplicate of https://github.com/flutter/flutter/issues/27956
